httpx/_utils.py

```diff
@@ -120,7 +120,7 @@ def guess_json_utf(data: bytes) -> typing.Optional[str]:
         if sample[1::2] == _null2:  # 2nd and 4th are null
             return "utf-16-le"
         # Did not detect 2 valid UTF-16 ascii-range characters
-    if nullcount == 3:
+    elif nullcount == 3:
```

Comment:
Either we should update all the cases here that could be `elif`, or none of them.
We're not really getting any value from the change, so I'd suggest we just leave it.

```suggestion
    if nullcount == 3:
```